batch7crud-roll1/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -0,0 +1,158 @@
+package com.lftechnology.batch7crud.controller;
+
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Employee;
+import com.lftechnology.batch7crud.service.EmployeeService;
+import com.lftechnology.batch7crud.util.TypeCaster;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class EmployeeController
+ */
+@WebServlet({"/employees/*"})
+public class EmployeeController extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+    EmployeeService employeeService = new EmployeeService();
+
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public EmployeeController() {
+        super();
+    }
+
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+	 */
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		
+		String requestInfo = request.getPathInfo();
+        System.out.println("requestInfo :" + requestInfo);
+
+        if(requestInfo == null){
+            list(request,response,1);
+        }else {
+
+            String[] pathParts = requestInfo.split("/");
+
+            if(pathParts[0].equals("") && pathParts[1].equals("create")){
+                create(request,response);
+            }else if(pathParts[2].equals("edit")){
+                int id = TypeCaster.toInt(pathParts[1]);
+                edit(request,response,id);
+            }
+        }
+	}
+
+	/**
+	 * @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse response)
+	 */
+	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)